src/legacy/core_plugins/kibana/public/home/components/sample_data_set_cards.js

```diff
@@ -181,9 +184,13 @@ export class SampleDataSetCards extends React.Component {
     });
   }
 
+  lightOrDarkImage = (sampleDataSet) => {
```

Comment:
Let's bug some engineers at some point about building some better services for this kind of stuff. I know we only have two themes, but doing this kind of boolean check for output won't scale long term. This has nothing to do with this PR, but I just want to write it out while we're thinking about it. I'd rather we had some sort of actual theme service that could load a default if the theme didn't have this setting (for example, a custom pink theme may not have an image). Better to start thinking about this kind of stuff now before it starts getting too out of hand. I'm worried we'll end up with a KUI problem, but in the JS layer, if we keep doing this over and over.

Reply:
Agreed

Reply:
To sort of tie this to how we do EUI stuff. There are themes, but the actual color functions don't check for a specific theme perse (aka the light or dark theme), they just adjust based upon the text color brightness that exists in those themes. It'd be nice to have something similar in JS logic because eventually, we'd like this all to be dynamic and arbitrary.